### ISSUE TITLE
[flutter-in-app-purchase] How can I run my License code in the debugger

### ISSUE THREAD

**Issue by ride4sun** [flutter/flutter#81799]
I did not find information how I can debug my license code. I know I can do a test release but I really want to debug the bloc I created.

**Comment by GZGavinZhao** [comment]
This is probably a question that would be better if put on Stack Overflow/Reddit/#help Discord channel as this is not a bug related to the Flutter framework or the `in-app-purchase` package itself. But thank you for posting this anyway! Please ask for some help using the methods above first before submitting an issue :) However, if you really think that there needs to be documentation about this topic, you might want to rename your suggestion as a documentation bug or an enhancement.

**Comment by pedromassangocode** [comment]
Hi @ride4sun 
This issue doesn't seem to describe a bug or a feature request. Please see https://flutter.dev/community for resources and asking questions like this,
you may also get some help if you post it on Stack Overflow and if you need help with your code, please see https://www.reddit.com/r/flutterhelp/
Closing, as this isn't an issue with Flutter itself. If you disagree, please write in the comments and I will reopen it.
Thank you

**Comment by ride4sun** [comment]
Well, I wasted another few days. The in_app_license package has many defects and I still don't see any description of how I can debug my code without releasing the package to google play alpha. Even do so, then I don't get any product information back and no errors etc. whats so ever. I am not sure how I should address that problem. Also, I see tons of open defects in the in_app_purchase package without much progress. Should be very important to google that people can mometize there work in a reliable fashion. 

I also searched stackoverflow and found this:
https://stackoverflow.com/questions/65303186/flutter-in-app-purchase-is-seriously-hard-to-test?noredirect=1#comment119154814_65303186

So - yes I am not sure where this belongs but it seems a bit to easy to just close it.
